public/components/datasources/components/__tests__/datasource.test.tsx

```diff
@@ -0,0 +1,39 @@
+/*
```

Comment:
Not sure how this got pulled in, should be on main already, can you double check/merge